packages/create-sitecore-jss/src/common/utils/helpers.ts

```diff
@@ -4,6 +4,18 @@ import path, { sep } from 'path';
 import { InitializerFactory } from '../../InitializerFactory';
 import { JsonObjectType } from '../processes/transform';
 
+// matched for proxy templates
+export const proxyAppMatcher = /node-.+-proxy/g;
+
+/**
+ * Returns the default path for proxy app initialized alongside main JSS tempalates.
+ * @param {string} mainAppDestination target destination for main app
+ * @param {string} proxyName name of for the proxy app folder
+ * @returns {string} target path for proxy app
+ */
+export const getDefaultProxyDestination = (mainAppDestination: string, proxyName: string) =>
```

Comment:
should we unit test this function for coverage?

Reply:
it returns a single value, the implementation is what you see, not sure if it needs a unit test